piet/src/gradient.rs

```diff
@@ -50,3 +54,158 @@ pub struct GradientStop {
     /// The color at that stop.
     pub color: Color,
 }
+
+/// A flexible, ergonomic way to describe gradient stops.
+pub trait GradientStops {
```

Comment:
You mentioned considering `Into<Vec<GradientStop>>`, was there a particular reason to prefer a new trait?

Reply:
Yes, you can't do `Into` on existing types, so the conversion from a slice into a Vec won't fly. With a new trait, no problem.